docs/10-Notes.md

```diff
@@ -85,6 +85,7 @@ There are many plugins that add additional functionality to Chart.js. Some parti
  - <a href="https://github.com/chartjs/Chart.Deferred.js" target="_blank">Chart.Deferred.js</a> - Defer initial chart update until chart scrolls into viewport
  - <a href="https://github.com/chartjs/Chart.smith.js" target="_blank">Chart.Smith.js</a> - Adds a smith chart type
  - <a href="https://github.com/chartjs/Chart.LinearGauge.js" target="_blank">Chart.LinearGauge.js</a> - Adds a linear gauge chart type
+ - <a href="https://github.com/00F100/Chart.Select2.js" target="_blank">Chart.Select2.js</a> - Use <a href="https://select2.github.io/" target="_blank">Select2</a> multiple select options do change the chart
```

Comment:
I totally agree, this kind of examples are very useful and should be well referenced (so thank you for sharing your experience :) ). I'm mainly saying that it should not be under the plugins section because it's not a plugin. Maybe a new links section ("How To", "Integrations", "Tutorials", etc.)? We should ask other @chartjs/maintainers what they think.

I [uploaded ans tweaked your example here](http://plnkr.co/edit/QT9epnu23kDxipNaEooQ?p=preview) and `chart.update()` works fine. Also thinking that it might be easier for people to interact with a fiddle, jsbin, codepen, plunker, etc., than having to clone a repository, install npm and run `npm install`.